src/main/java/co/inventrosoft/springboottask/repository/MatchRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,171 @@
+package co.inventrosoft.springboottask.repository;
+
+
+import co.inventrosoft.springboottask.configuration.JdbcConfig;
+import co.inventrosoft.springboottask.mapper.MatchMapper;
+import co.inventrosoft.springboottask.model.Match;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Repository;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Primary
+@Repository
+public class MatchRepositoryJdbcImpl implements MatchRepository{
+    private static final String UPDATE_QUERY = """
+        UPDATE matches SET tournament_id=?,
```

Comment:
Always try to make SQL easier to read (parse).
1) Here to find WHERE statement you should scan row by row to find in the end what you were looking for.
2) To find some specific column in select statement you also should scan row by row trying to separate columns by comma.

Of course your current query is not that big, and parse process does not take long time. But what will you do if you see complex query with 12 joins and 30 select columns with dozens of where conditions?

I propose structure that makes parse process faster.

```
UPDATE matches
SET tournament_id = ?,
    first_team_id = ?,
    ...
    round_code = ?,
    match_order = ?
WHERE match_id = ?;
```